mkosi/archive.py

```diff
@@ -119,6 +119,16 @@ def make_cpio(
     else:
         files = sorted(files)
 
+    # Filters out files that are below a symlink. If you have
+    # `lib/firmware/dirA/hugeblob.bin`, and `lib/firmware/dirB` is a
+    # symlink to `dirA`, we'd end up asking cpio to archive both
+    # `lib/firmware/dirA/hugeblob.bin` and
+    # `lib/firmware/dirB/hugeblob.bin`, i.e. two copies of the same
+    # file. Once Python 3.13 becomes the minimum, we can use
+    # the `recurse_symlinks=` argument to `Path.rglob()`,
```

Comment:
Can you make this
```suggestion
    # file. 
    # TODO: Once Python 3.13 becomes the minimum, we can use
    # the `recurse_symlinks=` argument to `Path.rglob()`,
```

`TODO` comments are much easier to grep for regularly.

Reply:
Good call, thanks.